hack/env

```diff
@@ -33,4 +33,10 @@
 # NOTE:   only committed code is built.
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
+relative_bin_path="$( os::util::repository_relative_path "${OS_OUTPUT_BINPATH}" )"
```

Comment:
Remember this is now going to download 1GB everytime you exit a container, so just be sure you want that pain.